WalletWasabi/Wallets/P2pBlockProvider.cs

```diff
@@ -99,7 +99,12 @@ public async Task<Block> GetBlockAsync(uint256 hash, CancellationToken cancellat
 					// Download block from selected node.
 					try
 					{
-						using (var cts = new CancellationTokenSource(TimeSpan.FromSeconds(RuntimeParams.Instance.NetworkNodeTimeout))) // 1/2 ADSL	512 kbit/s	00:00:32
+						// More permissive timeout if few nodes are connected to avoid exhaustion
+						var timeout = Nodes.ConnectedNodes.Count < 3
+							? Math.Min(RuntimeParams.Instance.NetworkNodeTimeout*1.5,600)
```

Comment:
Are you using CodeMaid? https://github.com/zkSNACKs/WalletWasabi/blob/23a068286ab696a9152f39ac148a1eeb1ea169f8/CONTRIBUTING.md#automatic-code-clean-up

Reply:
Please add some comments here - explain why.

Reply:
Yes but my issue (thanks @kiminuo btw) was that I had opened the folder and not the solution.
Now everything is working fine and I won't have this syntax glitches anymore